### PR TITLE
add foreman-plugins-el9 repoclosure config

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: package_manifest.yaml [hunk 1]
```diff
@@ -946,6 +946,17 @@ repoclosures:
...
+    plugins-staging-repoclosure-el9:
+      repoclosure_target_repos:
+        el9:
+          - "el9-foreman-plugins-{{ foreman_version }}-staging"
+      repoclosure_target_dist: el9
+      repoclosure_lookaside_repos:
+        el9:
+          - el9-baseos
+          - el9-appstream
+          - el9-puppet-7
```
Comment (evgeni) [inline package_manifest.yaml:958]: I didn't add salt, as we have no config for el9-salt yet.

https://repo.saltproject.io/salt/py3/redhat/9/ exists, but for some reason we've been using https://repo.saltproject.io/py3/redhat/ for other repos (note the missing "salt" in the folder structure)

**Comment by evgeni** [comment]
going to merge this, to unblock https://ci.theforeman.org/job/foreman-plugins-nightly-rpm-pipeline/

File: package_manifest.yaml (hunk 1 above)
Comment (ekohl) [reply package_manifest.yaml:958]: https://docs.saltproject.io/salt/install-guide/en/latest/topics/install-by-operating-system/centos.html all uses https://repo.saltproject.io/salt/py3/redhat/ so I think we should switch over.

Comment (evgeni) [reply package_manifest.yaml:958]: https://github.com/theforeman/foreman-packaging/pull/10219

**Pull request merged**